src/server/game/AI/NpcBots/botcommands.cpp

```diff
@@ -4986,26 +4986,29 @@ class script_bot_commands : public CommandScript
     {
         Player* owner = handler->GetSession()->GetPlayer();
         Unit* u = owner->GetSelectedUnit();
+
+        auto ReviveAllFor = [&](Player* master) -> bool
+            {
+                if (!master->HaveBot())
+                {
+                    handler->PSendSysMessage("{} has no npcbots!", master->GetName());
+                    handler->SetSentErrorMessage(true);
+                    return false;
+                }
+
+                master->GetBotMgr()->ReviveAllBots();
+                handler->SendSysMessage("Npcbots revived");
+                return true;
+            };
+
         if (!u)
         {
-            handler->SendSysMessage(".npcbot revive");
-            handler->SendSysMessage("Revives selected npcbot. If player is selected, revives all selected player's npcbots");
-            handler->SetSentErrorMessage(true);
-            return false;
+            return ReviveAllFor(owner);
```

Comment:
Please do not make a command revive everything when nothing is selected. We need to print help somewhere and this is the best place for it.